src/pages/ServiceDetails/ServiceDetailsPage.tsx

```diff
@@ -191,22 +195,24 @@ class ServiceDetails extends React.Component<RouteComponentProps<ServiceId>, Ser
     let to = '/namespaces/' + this.props.match.params.namespace + '/services/' + this.props.match.params.service;
     return (
       <>
-        <div className="page-header">
-          <h2>
-            Service{' '}
+        <Breadcrumb title={true}>
```

Comment:
I think here I would do the same with the breadcrumb generation. Thinking in OO, I would tend to user render as a distributor, placing the layout and calling to internal calls. Object it would be the view, and methods would be: breadcrumb, header, table of whatever, and so on. IMO, this way we can avoid long render methods and will help to have better testing for each view.

Reply:
I am afraid that when per breadcrumb you should add an onClick, splitting will add more complexity, so I decided to placed in the same location where it was the previous `h2` "breadcrumb".